package.nls.json

```diff
@@ -1,112 +0,0 @@
-{
```

Comment:
Why delete this?

Reply:
There was a high amount of changes/adds/deletes to the strings so the old `package.nls.json` was out of date. I felt it's easier to just create new one (rather than diff it) in this situation. After we've stabilized on functionality, it will be recreated.

Reply:
Read the PR description :) leaving this comment here as a TODO reminder then

---
In reply to: [430734644](https://github.com/microsoft/sarif-vscode-extension/pull/279#discussion_r430734644) [](ancestors = 430734644)

Reply:
And I'll do it does Gabe doesn't have to do it again. :-)

Reply:
Tracking this as an issue ([https://github.com/microsoft/sarif-vscode-extension/issues/281](https://github.com/microsoft/sarif-vscode-extension/issues/281)) and resolving this thread.

---
In reply to: [430756276](https://github.com/microsoft/sarif-vscode-extension/pull/279#discussion_r430756276) [](ancestors = 430756276,430734644)